Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml

```diff
@@ -1014,7 +1106,14 @@ inputs:
   required: false
   description: A comma-separated list of destination zones.
   playbookInputQuery:
+- key: SecondaryDeviceGroup
+  value: {}
+  required: false
+  description: In that case that the rule, address and service are created in the "Shared" location, we need to know what device-groups we can push to because it isn't possible to push to the "Shared" location.
```

Comment:
```suggestion
  description: If the rule, address and service are created in the "Shared" location, we need to know what device-groups we can push to because it isn't possible to push to the "Shared" location.
```